packages/release-please/test/release-please.ts

```diff
@@ -339,16 +346,17 @@ describe('ReleasePleaseBot', () => {
       );
       const requests = nock('https://api.github.com')
         .get(
-          '/repos/Codertocat/Hello-World/contents/.github/release-please.yml'
+          '/repos/Codertocat/Hello-World/contents/.github%2Frelease-please.yml'
         )
-        .reply(200, {content: config.toString('base64')})
+        .reply(200, config)
         .delete(
-          '/repos/Codertocat/Hello-World/issues/2/labels/release-please:force-run'
+          '/repos/Codertocat/Hello-World/issues/2/labels/release-please%3Aforce-run'
         )
         .reply(200);
 
       await probot.receive({
-        name: 'pull_request.labeled',
+        // see: https://github.com/probot/probot/issues/1367
+        name: 'pull_request.labeled' as any,
```

Comment:
Remove `.labeled` from the name. `labeled` is the action, it is part of the payload: https://github.com/googleapis/repo-automation-bots/blob/e9637b11eedb2fdecf746e99dffbee347bd052a0/packages/release-please/test/fixtures/pull_request_labeled.json#L2

```suggestion
        name: 'pull_request',
```